features/front_office/renewals/renewal_convictions_checks.feature

```diff
@@ -4,20 +4,22 @@ Feature: Registered waste carrier declares conviction during renewal
   I want to be informed of any potential matches with the Environment Agencies convictions database
   So that I can then target my investigation of registration suitability to the right people
 
+@email
```

Comment:
Just alignment of the `@email`

Reply:
Aligned with the scenario, if that's what you meant?